vale/main.py

```diff
@@ -45,7 +46,10 @@ def get_target() -> (str, str, str):
     elif sys.platform.startswith("win32"):
         operating_system = "Windows"
 
-    if os.uname().machine.startswith("x86_64"):
+    if operating_system == "Windows":
+        convert_arch = {"32bit": "32-bit", "64bit": "64-bit"}
+        architecture = convert_arch.get(platform.architecture()[0], None)
```

Comment:
I tested this and it actually works with Linux as well! I'll accept this fix and apply it also for Linux, it seems a more reliable procedure. :top:

Reply:
Right, I tried to keep the edits as minimal as possible for now, but you should be able to use the platform module to get all the info you need, and it would probably be a bit neater than the windows specific if statement I added.